### ISSUE TITLE
Test Pattern Selection is not greyed out and is available for use.

### ISSUE THREAD

**Issue by qbancoffee** [shuuryou/imacg3#9]
"Test Pattern Selection" is not greyed out and is available for use before "Show Test Pattern" has been
enabled.

If a test pattern is selected the application crashes.

However, if "show test pattern" is first enabled and then disabled then "Test Pattern Selection" is
greyed out and can't be enabled as it should.

**Comment by shuuryou** [comment]
The test pattern menu item wasn't disabled initially. I just fixed this. I think this creeped in when I noticed Mono doesn't honor the CheckOnClick property of menu items. It should work like this:

* Test pattern selector menu is unavailable/grayed out at launch
* Test pattern is activated and covers the screen
* Test pattern selector menu becomes available
* Test pattern is dismissed via menu
* Test pattern selector menu becomes unavailable again
